pyaedt/application/MessageManager.py

```diff
@@ -117,23 +127,33 @@ def __init__(self, parent=None, loadondesktop=True, loadonlog=True):
 
     @property
     def messages(self):
-        """Return the AEDT Message Manager Contents for the active project and design
+        """Message manager content for the active project and design.
 
         Returns
         -------
-        MessageList
+        list
+           List of messages.
         """
         return self.get_messages(self._project_name, self._design_name)
 
     @aedt_exception_handler
     def get_messages(self, project_name, design_name):
-        """Return the AEDT Message Manager Contents for a specified project and design
-
-        If the design- or project names are invalid, they will be ignored
+        """Retrieve the Message Manager content for a specified project and design.
+        
+        Parameters
+        ----------
+        project_name: str,
+            Name of the project.
+        design_name, str
+            Name of the design within the specified project.
+        
+        ..note::
+          If the design or project names are invalid, they are ignored.
```

Comment:
```suggestion
        .. note::
           If the design or project names are invalid, they are ignored.
```

Reply:
Always a space after `..` for note and warning directives.  The standard indentation here is three spaces (rst standard).